### PR TITLE
13.0 mig l10n ch states

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Work done:
1. Set new version on `__manifest__`
2. Squash OCA commits to previous commit
Questions: 
1. Is it correct that I am now part of every commit?. I see also this on your PR #516 
2. Something that I still don’t understand is why Germany and France are states of Switzerland.. But it has been migrated version after version.. Maybe somebody can explain me why

### CODE REVIEW

**Comment by yvaucher** [comment]
thanks @pegonzalezspesol 

I will wait that 13.0 is ready for PR.

https://github.com/OCA/maintainer-quality-tools/pull/613
I want pre-commit hook is installed before accepting any module.

Which should be a matter of few days.

**Comment by yvaucher** [comment]
> Is it correct that I am now part of every commit?. I see also this on your PR #516

You are not set as the author but the one who pushed the commits. This is normal.

**Comment by yvaucher** [comment]
Can you squash the 3 first commits about EOL

2019-10-11-131312_692x99_scrot


+ squash the 2 "Update de.po" commits

2019-10-11-131319_694x72_scrot

**Comment by pegonzalezs** [comment]
@yvaucher squash done

**Comment by yvaucher** [comment]
@pegonzalezs pre-commit has now be installed on this repository. Pleas rebase.

> pre-commit has been adopted by the OCA and installed to this repository.
> All 13.0 branch of the OCA now comes with pre-commit hook.
> 
> You might have to run `pre-commit run -a` to auto-fix what needs to be fixed.
> 
> For further development on branch 13.0 on any OCA repository pre-commit hooks will be available and can be installed with `pip install pre-commit` then `pre-commit install`.

**Comment by i-vyshnevska** [comment]
@pegonzalezspesol  you have an extra input in https://github.com/OCA/l10n-switzerland/pull/520/commits/bfc2b1574ba8035372b70b176d31731a0554e257?file-filters%5B%5D=.py&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile 
can you rebase and check it again

File: .isort.cfg [hunk 1]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=
+known_third_party=setuptools
```
Comment (i-vyshnevska) [inline .isort.cfg:12]: this change shouldn't be here

Comment (pegonzalezspesol) [reply .isort.cfg:12]: absolutely @i-vyshnevska .. but then the travis checks fails.. did you try to commit with current 13.0/travis configuration?

Comment (i-vyshnevska) [reply .isort.cfg:12]: ok, push this in separate commit pls

Comment (i-vyshnevska) [reply .isort.cfg:12]: @pegonzalezspesol  sorry, I wasn't very explicit, 
that change about issort is correct, you can open other PR or push in this but is separate commit so migration commits are not mixup with travis setup

Comment (pegonzalezspesol) [reply .isort.cfg:12]: done @i-vyshnevska

**Comment by pegonzalezspesol** [comment]
@i-vyshnevska .. do you know what's the problem in the continious integration? Checking the "Full Logs" seems normal

**Review by i-vyshnevska — APPROVED** [review]

**Comment by i-vyshnevska** [comment]
> @i-vyshnevska .. do you know what's the problem in the continious integration? Checking the "Full Logs" seems normal

sometimes happens, not blocking for merging

**Comment by yvaucher** [comment]
Runbot shows an error because it has warnings.

This is unrelated to this PR.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**